test/addressSpec.js

```diff
@@ -0,0 +1,146 @@
+var addr = require('../src/js/address');
+
+describe('address functions', function() {
+  describe('is adddress duplicate', function() {
+    var address,
+      duplicates;
+
+    beforeEach(function(){
+      address = '123 Main St., Somecity, SomeWhere, 12345';
```

Comment:
Shouldn't need address in the beforeEach, since it will stay the same value for each test.